src/styles/layout/index/catalog.scss

```diff
@@ -53,3 +48,94 @@
     background: var(--card-background);
   }
 }
+
+@include laptop {
+  .catalog {
+    &__list {
+      display: grid;
+      grid-auto-flow: column;
+      grid-auto-columns: calc((100% - 72px) / 4);
```

Comment:
Interesting solution and I like your `calc`, but could you please clarify what happens here and could we do it without `calc`?